integration/bin/tachyon-yarn-setup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# prepare Tachyon dir structure for a YARN container to launch master or worker
```

Comment:
Maybe add a comment saying that, this script is referenced as 
`tachyon.yarn.Utils#TACHYON_SETUP_SCRIPT` in source code.
So in the future if we rename this script, we should also update the source code.